syne_tune/optimizer/schedulers/searchers/bayesopt/tuning_algorithms/defaults.py

```diff
@@ -25,3 +25,5 @@
 DEFAULT_NUM_INITIAL_CANDIDATES = 250
 
 DEFAULT_NUM_INITIAL_RANDOM_EVALUATIONS = 3
+
+DEFAULT_MAX_SIZE_DATA_FOR_MODEL = 500
```

Comment:
According to experiments on NB201-CIFAR10 and NB301 with 8 workers and long max_wallclock_time, this works as well as larger values or no limit.